src/fmeval/eval_algorithms/helper_models/helper_model.py

```diff
@@ -100,12 +100,17 @@ class DetoxifyHelperModel(BaseHelperModel):
     """
     Helper model for Detoxify: https://github.com/unitaryai/detoxify
 
+    Note: we load the unbiased model directly from the state dict due to dependency conflicts between detoxify and
+    transformers libraries.
+
     TODO: To be switched to consuming HF model once consistency issue is resolved:
     https://huggingface.co/unitary/unbiased-toxic-roberta. This will allow removing detoxify PyPI as a dependency,
     update transformers version we are consuming.
     """
 
-    DETOXIFY_MODEL_TYPE = "unbiased"
+    UNBIASED_MODEL_URL = (
+        "https://github.com/unitaryai/detoxify/releases/download/v0.3-alpha/toxic_debiased-c7548aa0.ckpt"
```

Comment:
Should we add this file directly to the `fmeval` repo so that we don't rely on the detoxify repo? This isn't a major concern.

Reply:
Let's discuss this with science team first, if we need to check on it with legal.